src/containers/SaveBarHandler/SaveBarButton/index.jsx

```diff
@@ -1,19 +1,24 @@
 import styles from '../style.postcss';
 
 import React from 'react';
+import classnames from 'classnames';
 import Button from 'components/Button';
 import PropTypes from 'prop-types';
+import testClass from 'domain/testClass';
 
-const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) =>
-  <Button autoWidth
+const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) => {
+  const className = classnames(styles.SaveBar_button,
+    testClass(`${label.replace(/\s+/g, '')}-btn`));
```

Comment:
Let's try to make it nicer than this. `test-DoSomething-btn` or `test-Dosomething-btn`, etc. would look horrible.

We can replace spaces with dashes `-`, and make sure everything is lowercase with something like `label.replace(/\s+/g, '-').toLowerCase()`.